### PR TITLE
Custom Sidebar logo

### PR DESCRIPTION
**Pull request by mnholtz**
## What does this PR do?

- Extension-side implementation of https://github.com/pixiebrix/pixiebrix-app/issues/2422
- This adds a custom Sidebar logo to the Sidebar that can be configured as part of an Organization theme. A custom Organization logo will override a Partner logo, i.e. if the user's Organization has a custom logo, the Automation Anywhere logo will get replaced.
- Note that if a user does have a Partner theme applied, those theme colors will still be applied at this time, following behavior from https://github.com/pixiebrix/pixiebrix-extension/pull/4467

## Reviewer Tips

- Primary change was the caching/sidebar logo addition to the `useOrganizationTheme` hook in `useTheme.ts`. The logo returned from this hook was then added to `Header.ts`.
- Swagger file updates are mostly unrelated to this PR. The only significant change there is the addition of the `logo` field on `OrganizationTheme`.

## Discussion

- Note in the demo that while we are caching the Organization theme, there will a flicker the first time the cache is updated. This is unavoidable (as far as I know) as long as we want to optimistically apply the theme.

## Demo

Loom: https://www.loom.com/share/342b668e02944113a2ade70cf3d70039

https://user-images.githubusercontent.com/36575242/196291450-3b280f22-48c5-4213-ba00-86d83f59e17f.mov

## Future Work

- We might want to revisit https://github.com/pixiebrix/pixiebrix-extension/issues/4405 to see if there's anything left to do (I think there are some outstanding caching problems with the Sidebar color flickering introduced in https://github.com/pixiebrix/pixiebrix-extension/pull/4467). I realized that we already got theme caching for free in the case of the Sidebar logo.

## Checklist

- [x] Add tests
- [x] Run Storybook and manually confirm that all stories are working
- [x] Designate a primary reviewer @BLoe 
bloom

### CODE REVIEW

**Review by BLoe — APPROVED** [review]
LGTM just one comment 🚀

File: src/hooks/useTheme.ts [hunk 1]
```diff
@@ -89,22 +89,33 @@ export function useGetTheme(): Theme {
...
-  const organizationTheme = me?.organization?.theme;
+  const { organization: cachedOrganization } = useSelector(selectAuth);
+
+  const organizationTheme = useMemo(() => {
+    if (me) {
+      return me?.organization?.theme;
+    }
+
+    return cachedOrganization?.theme;
+  }, [me, cachedOrganization]);
```
[the remark this answers is not in the document]
Comment (fregante) [reply src/hooks/useTheme.ts:105]: From what I understand, we shouldn't use a hook either for derived data. See last code block on this page https://tkdodo.eu/blog/dont-over-use-state

**Comment by fregante** [comment]
@mnholtz I pushed an unrelated fix as I suggested on Slack, see https://github.com/pixiebrix/pixiebrix-extension/pull/4498#issuecomment-1284134015

File: src/hooks/useTheme.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (mnholtz) [reply src/hooks/useTheme.ts]: I'll try to get better at breaking this habit 😅

**Pull request merged**